elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -927,8 +932,10 @@ private boolean interfaceHasFilterExpression(Class<?> entityInterface) {
         }
 
         final Class<?> relationClass = dictionary.getParameterizedType(obj, relationName);
-        if (! requestScope.getPagination().isDefaultInstance()
-                && !CanPaginateVisitor.canPaginate(relationClass, dictionary, requestScope)) {
+        boolean isAnyToMany = dictionary.isMappedInterface(relationClass)
+                && dictionary.getAttributeOrRelationAnnotation(obj.getClass(), ToMany.class, relationName) != null;
```

Comment:
Are there any ways for us to use `MappedInterface`s and not have them marked `AnyToMany`? If so, this could be an overly restrictive check, right?

Reply:
This is correct, I will fix it.